plugincontainer/go.mod

```diff
@@ -1,42 +1,52 @@
 module github.com/hashicorp/go-secure-stdlib/plugincontainer
 
-go 1.20
+go 1.21
 
 require (
-	github.com/docker/docker v24.0.7+incompatible
+	github.com/docker/docker v26.1.0+incompatible
```

Comment:
We'll probably want to bump this v26.1.5 to avoid the [latest docker CVE](https://github.com/moby/moby/security/advisories/GHSA-v23v-6jw2-98fq).